google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryParameterValue.java

```diff
@@ -254,6 +255,14 @@ public static QueryParameterValue string(String value) {
     return of(value, StandardSQLTypeName.STRING);
   }
 
+  /**
+   * Creates a {@code QueryParameterValue} object with a type of JSON. This is only supported in
```

Comment:
Please add: `Currently` ... this is only supported in INSERT

Reply:
Done.